lib/tasks/rotate.rake

```diff
@@ -25,12 +24,13 @@ namespace :rotate do
   desc 'encrypt plain OTP secret key'
   task encrypt_otp: :environment do
     num_users = User.where.not(otp_secret_key: nil).count
-    progress = new_progress_bar('Users', num_users)
```

Comment:
the unused variable was refactored away, but the call to `new_progress_bar` is still there (Below)